core/testdata/testspecs/v2_specs.go

```diff
@@ -124,6 +124,7 @@ type KeeperSpecParams struct {
 	ContractAddress string
 	FromAddress     string
 	EvmChainID      int
+	Confirmations   int
```

Comment:
Is it incoming or outgoing?

`MinRequiredIncomingConfirmations` or `MinIncomingConfirmations` would be clearer